kinase_focused_fragment_library/ligand_analysis/base.py

```diff
@@ -13,6 +13,7 @@
 import pandas as pd
 from rdkit import Chem, DataStructs
 from rdkit.Chem import PandasTools, rdFingerprintGenerator
+import wandb 
```

Comment:
I'm not sure if I would add wandb in the final version. If we add it, we should also add it to the environment and add a note somewhere maybe? Also, do you need a wandb account to run the code then?

Reply:
Oh, that was accidentally. Thanks :)